controllers/articles.controller.js

```diff
@@ -21,5 +21,18 @@ function getArticles(req, res, next) {
 
 }
 
+function patchArticle(req, res, next) {
 
-module.exports = { getArticle, getArticles }
\ No newline at end of file
+    const { article_id } = req.params
+    const { inc_votes } = req.body
+
+    updateArticle(article_id, inc_votes)
+        .then((article) => {
+            res.status(200).send({ article: article })
```

Comment:
Great that you're sending this through on a clearly labeled key
You could use shorthand here as the key has the same name as the variable
ie. `{ article: article }`
is equivalent to
`{ article }`